translations/hu/get5.phrases.txt

```diff
@@ -0,0 +1,379 @@
+    }
+    "StopCommandNotEnabled"
+    {
+        "hu"            "A stop parancs nincs engedélyezve."
+    }
+    "BackupLoadedInfoMessage"
+    {
+        "hu"            "Sikeresen visszatöltődött a körmentés {1}"
+    }
+    "MatchBeginInSecondsInfoMessage"
+    {
+        "hu"            "A mérkőzés {1} másodperc múlva kezdődik."
+    }
+    "MatchIsLiveInfoMessage"
+    {
+        "hu"            "A mérkőzés {GREEN}ÉLES"
+    }
+    "KnifeIn5SecInfoMessage"
+    {
+        "hu"            "A kés kör 5 másodperc múlva kezdődik."
+    }
+    "KnifeInfoMessage"
+    {
+        "hu"            "Kés!"
+    }
+    "TeamDecidedToStayInfoMessage"
+    {
+        "hu"            "{1} úgy döntött, hogy maradnak."
+    }
+    "TeamDecidedToSwapInfoMessage"
+    {
+        "hu"            "{1} úgy döntött, hogy térfelet cserélnek."
+    }
+    "TeamLostTimeToDecideInfoMessage"
+    {
+        "hu"            "{1} maradnak, mivel idő letelte előtt, nem választottak."
+    }
+    "ChangingMapInfoMessage"
+    {
+        "hu"            "Pályaváltás a következőre {GREEN}{1}..."
+    }
+    "MapDecidedInfoMessage"
+    {
+        "hu"            "A pályák ellettek döntve:"
+    }
+    "MapIsInfoMessage"
+    {
+        "hu"            "{1}. Pálya: {GREEN}{2}{NORMAL}"
+    }
+    "TeamPickedMapInfoMessage"
+    {
+        "hu"            "{1} választotta a {GREEN}{2} {NORMAL} {3}. pályának."
+    }
+    "TeamSelectSideInfoMessage"
+    {
+        "hu"            "{1} választott, hogy kezd {GREEN}{2} {NORMAL} oldalon a {3}. pályán"
+    }
+    "TeamVetoedMapInfoMessage"
+    {
+        "hu"            "{1} szavazott {LIGHT_RED}{2}{NORMAL}."
+    }
+    "CaptainLeftOnVetoInfoMessage"
+    {
+        "hu"            "A csapatkapitány kilépett a vétó során, a vétó szünetel."
+    }
+    "ReadyToResumeVetoInfoMessage"
+    {
+        "hu"            "Írd be a {GREEN}!ready {NORMAL}parancsot ha készenállsz a vétó folytatására."
+    }
+    "MatchConfigLoadedInfoMessage"
+    {
+        "hu"            "Mérkőzés beállítás betöltödött."
+    }
+    "MoveToCoachInfoMessage"
+    {
+        "hu"            "Mivel a csapatod megtelt, így edzői poziccióba lettél átmozgatva."
+    }
+    "ReadyTag"
+    {
+        "hu"            "[FELKÉSZÜLT]"
+    }
+    "NotReadyTag"
+    {
+        "hu"            "[NEM ÁLL KÉSZEN]"
+    }
+    "MatchPoweredBy"
+    {
+        "hu"            "{YELLOW}Get5{NORMAL} által megvalósítva"
+    }
+    "MapVetoPickMenuText"
+    {
+        "hu"            "Válassz pályát, amin játszanál:"
+    }
+    "MapVetoPickConfirmMenuText"
+    {
+        "hu"            "Megerősíti, hogy {1} játszana?:"
+    }
+    "MapVetoBanMenuText"
+    {
+        "hu"            "Válaszon egy pályát amit vétózna:"
+    }
+    "MapVetoBanConfirmMenuText"
+    {
+        "hu"            "Megerősíti, hogy {1} vétózza?:"
+    }
+    "MapVetoSidePickMenuText"
+    {
+        "hu"            "Válasszon oldalt {1}:"
+    }
+    "MapVetoSidePickConfirmMenuText"
```

Comment:
Confirm you want to start {1}:
1. Yes
2. No

Because there are Yes or No so it's better in Hungarian to ask the person does he/she want to confirm his/her choice?
It depends on the person, but yeah it could be only Confirmation on his/her choice without question

Reply:
It's just to follow the pattern. You could make the same argument for English. It's not really meant to be understood as a question, and the combination of colon and question mark looks odd to me.

Reply:
If we add the question mark, we should do it consistently across all languages and then remove the colon. One or the other. You can decide for Hungarian, but don't combine question mark and colon.